docs/releases.md

```diff
@@ -30,6 +30,17 @@
 
 ## Master branch
 
+Lychee currently undergoes a major architectural refactoring of its internals.
+The master branch already includes the first part of that refactoring which mostly concentrates on the backend.
+
+These changes and information only affect those users who directly follow the master branch:
+
+- Despite the substantial changes your photos and albums should be as safe as with any other commit to the master branch. Don't worry.
+- As we did our best to ensure that Lychee behaves as before, you might potentially encounter more regression bugs than usually. These bugs are _not_ expected to severely affect the overall functioning of Lychee, but may be annoying.
```

Comment:
I'm not clear what message we are trying to convey with this sentence, so the below suggestion is just my guess:

_While we did our best to ensure that Lychee behaves as before, the large amount of recent changes means that you may encounter more regression bugs than usually. Please make sure to test the functionality you depend on and promptly report any regressions you observe; we will do our best to address any such bugs as quickly as possible._

Reply:
I like @kamil4's version, but `s/usually/usual/`